### PR TITLE
Translate GuidePanel and HelpText, remove text from InternalHeader.

### PR DESCRIPTION
**Pull request by HalvorHaugan**
### Component Checklist 📝

- [ ] JSDoc
- [ ] Examples
- [ ] Documentation / Decision Records
- [x] Storybook
- [ ] Style mappings (`@navikt/core/css/config/_mappings.js`)
- [ ] Component tokens (`@navikt/core/css/tokens.json`)
- [ ] CSS class deprecations (`@navikt/aksel-stylelint/src/deprecations.ts`)
- [ ] Exports (`@navikt/core/react/src/index.ts` and `@navikt/core/react/package.json`)
- [ ] New component? CSS import (`@navikt/core/css/index.css`)
- [ ] Breaking change? Update migration guide. Consider codemod.
- [x] Changeset (Format: `<Component>: <gitmoji?> <Text>.` E.g. "Button: :sparkles: Add feature xyz.")

### CODE REVIEW

File: .changeset/fair-countries-admire.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@navikt/ds-react": patch
+---
+
+InternalHeader.UserButton: Remove `title` on icon and fix types
```
Comment (KenAJoh) [inline .changeset/fair-countries-admire.md]:
```suggestion
InternalHeader.UserButton: Remove `title`-attribute on icon and improved types.
```

File: @navikt/core/react/src/util/i18n/locales/en.ts [hunk 2]
```diff
@@ -123,4 +123,10 @@ export default {
     maxSelected: "{selected} of max {limit} are selected.",
     clear: "Clear",
   },
+  GuidePanel: {
+    illustrationLabel: "Illustration of advisor",
+  },
+  HelpText: {
+    title: "Help",
```
Comment (KenAJoh) [inline @navikt/core/react/src/util/i18n/locales/en.ts]: Just "Help" seems a little ambiguous if not used inside a specific context. I see Skatteetaten.no also use "help", so might not be an actual issue and we have used this for 6+ years now without issue.
Could any of these work?
- Help description
- Help text
- Help content
- Additional information
- Supplemental information

Comment (HalvorHaugan) [reply @navikt/core/react/src/util/i18n/locales/en.ts]: Agreed. I think I'll go for an "information" variant. What do you think about just "More information"? "Additional" feels like a more "complex" word than necessary, if that makes sense. Or is there a difference between these words that's relevant here?

Comment (KenAJoh) [reply @navikt/core/react/src/util/i18n/locales/en.ts]: "More information" works 👍

**Review by KenAJoh — APPROVED** [review]
👏 🚢 🇮🇹

**Pull request merged**